.build/azure-pipelines-example-multistage.yml

```diff
@@ -48,8 +48,8 @@ jobs:
       downloadPath: '$(System.ArtifactsDirectory)'
 
   - script: |
-      curl -o run_infersharp.sh https://raw.githubusercontent.com/microsoft/infersharpaction/v1.4/run_infersharp_ci.sh
-      curl -o .inferconfig https://raw.githubusercontent.com/microsoft/infersharp/v1.4/.inferconfig
+      curl -o run_infersharp.sh https://raw.githubusercontent.com/microsoft/infersharpaction/v1.4.1/run_infersharp_ci.sh
```

Comment:
InferSharpAction introduced a workaround on SARIF display on GitHub Security. v1.4.1 has the same content as v1.5. Will revisit this if future versions start to differ.